release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -456,7 +461,7 @@ For more information, see xref:../updating/preparing_for_updates/updating-cluste
 === Networking
 
 [id="ocp-4-19-networking-support-load-secrets_{context}"]
-==== Creating a route with externally managed certificate (General Availability) 
+==== Creating a route with externally managed certificate (General Availability)
```

Comment:
```suggestion
==== Creating a route with an externally-managed certificate (General Availability)
```
The hyphen might be excluded in headings, but adding it here for your consideration.
ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-hyphens#compound-words